psd-web/db/migrate/20200207112514_create_user_roles.rb

```diff
@@ -0,0 +1,9 @@
+class CreateUserRoles < ActiveRecord::Migration[5.2]
+  def change
+    create_table :user_roles do |t|
+      t.belongs_to :user, type: :uuid
+      t.string :name
```

Comment:
Might be worth preventing nulls in these columns (even though these should get caught by the validation):

```suggestion
      t.string :name, null: false
```